packages/client/src/clients/in-app-messages/channel-client.ts

```diff
@@ -16,19 +18,30 @@ import {
 export class InAppChannelClient {
   public store: InAppStore;
 
+  private socketDriver: InAppMessageSocketDriver | undefined;
```

Comment:
Open to a different name if anyone has better suggestions.

Reply:
I think `InAppMessageSocketManager` is better than `Driver`

Reply:
Yep that works too, will update.